spec/index.html

```diff
@@ -533,6 +542,340 @@ <h3>Overview</h3>
+            <tr>
+              <td><var>temporary issuer</var> mappings</td>
+              <td>
+                <table class="hash-related">
+                  <thead><tr>
+                    <th><var>original</var></th>
+                    <th><var>canonical</var></th>
+                  </tr></thead>
+                  <tbody>
+                    <tr><td>`_:e0`</td><td>`_:b0`</td></tr>
+                  </tbody>
+                </table>
+              </td>
+            </tr>
+            <tr>
+              <td><var>hash</var></td>
+              <td>`fbc300de5afafd97a4b9ee1e72b57754dcdcb7ebb724789ac6a94a5b82a48d30`</td>
+            </tr>
+          </tbody>
+        </table>
+
+        <table id="ex-ca-shared-result-e1"
+               class="hash-related numbered">
+          <caption>Result for `_:e1`</caption>
+          <tbody>
+            <tr>
+              <td><var>temporary issuer</var> mappings</td>
+              <td>
+                <table class="hash-related">
+                  <thead><tr>
+                    <th><var>original</var></th>
+                    <th><var>canonical</var></th>
+                  </tr></thead>
+                  <tbody>
+                    <tr><td>`_:e1`</td><td>`_:b0`</td></tr>
+                  </tbody>
+                </table>
+              </td>
+            </tr>
+            <tr>
+              <td><var>hash</var></td>
+              <td>`2c0b377baf86f6c18fed4b0df6741290066e73c932861749b172d1e5560f5045`</td>
+            </tr>
+          </tbody>
+        </table>
+
+        <p><a href="#ca-5-3">Step 5.3</a> creates
+          the canonical identifiers for the temporary identifiers
+          established in the previous step
+          running in order of the <var>hash</var> component from each result.
+          This updates the <a>canonical issuer</a> with the following
+          mappings:</p>
+
+        <table id="ex-ca-shared-canon-identifiers"
+               class="hash-related numbered">
+          <caption>Original to canonical blank node identifiers</caption>
+          <thead><tr>
+            <th><var>original</var></th>
+            <th><var>canonical</var></th>
+          </tr></thead>
+          <tbody>
+            <tr><td>`_:e0`</td><td>`_:c14n3`</td></tr>
+            <tr><td>`_:e1`</td><td>`_:c14n2`</td></tr>
+            <tr><td>`_:e2`</td><td>`_:c14n0`</td></tr>
+            <tr><td>`_:e3`</td><td>`_:c14n1`</td></tr>
+          </tbody>
+        </table>
+
+        <p><a href="#ca-6">Step 6</a> generates
+          the normalized dataset by replacing blank node identifiers in the original
+          input with their canonical identifiers:</p>
+
+        <pre id="ex-ca-normalized-shared-dataset" data-transform="updateExample">
+          <!--
+            :p :q _:c14n2 .
+            :p :q _:c14n3 .
+            _:c14n0 :r _:c14n1 .
+            _:c14n2 :p _:c14n1 .
+            _:c14n3 :p _:c14n0 .
+          -->
+        </pre>
+      </aside>
+    </section>
```

Comment:
I haven't run this example through any of our own implementations to confirm it, but don't want to hold up the PR because of that. I know @gkellogg has generated this using an implementation that passes the same test suite that our implementations pass, so I suspect it's either correct or that we'd quickly find consensus on where any fixes are needed.